packages/accordion/README.md

```diff
@@ -1,13 +1,7 @@
 # [`@fremtind/jkl-accordion`](https://fremtind.github.io/jokul/components/accordion/)
 
-## Beskrivelse
+# Trekkspillister
+Trekkspillister lar oss presentere mer innhold på liten plass ved å vise informasjonen vise og skjule innhold selv. En trekkspilliste består av en eller flere rader som kan trykkes på for å vise eller skjule informasjon.
```

Comment:
Trekkspillister lar oss presentere mer innhold på liten plass ved å ~~vise informasjonen~~ vise og skjule innhold selv.

Reply:
_Forslag til forenkling:_
Trekkspillister lar oss presentere mye innhold på liten plass. En trekkspilliste består av en eller flere rader som kan trykkes på for å vise eller skjule informasjon.